relays/list_transform.ts

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+import { Message } from "mhub";
+
+/**
+ * @param msg Message just received on this binding.
+ * @return Message(s), promise for message(s), or void
+ */
+export default function(msg: Message): void|Message|Message[]|Promise<Message> {
+	if (msg.topic === "scores:ranking") {
+		let result = msg.clone();

```

Comment:
Your new message doesn't look at all like the old message, so cloning and then overwriting all properties seems confusing. I think this would be more logical:

```ts
const data = {
    header: msg.data.stage.name,
    data: msg.data.ranking.map(/* rest of map here */)
};
return new Message("list:setArray", data);
```